readme.md

```diff
@@ -2,4 +2,6 @@
 
 [![Automated E2E Testing](https://github.com/haiderf7/social-media-client/actions/workflows/e2e-test.yml/badge.svg)](https://github.com/haiderf7/social-media-client/actions/workflows/e2e-test.yml)
 
+[![Code Review](https://github.com/haiderf7/social-media-client/actions/workflows/gpt.yml/badge.svg)](https://github.com/haiderf7/social-media-client/actions/workflows/gpt.yml)
```

Comment:
There is more you can add to this readme. See this template for ref https://gist.github.com/Igormandello/57d57ee9a9f32a5414009cbe191db432